core/designsystem/src/main/java/com/dev/innerview/core/designsystem/component/Dialog.kt

```diff
@@ -0,0 +1,142 @@
+package com.dev.innerview.core.designsystem.component
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import com.dev.innerview.core.designsystem.theme.InnerViewTheme
+import com.dev.innerview.core.designsystem.theme.Paddings
+
+@Composable
+fun InnerViewDialog(
+    titleText: String = "",
+    contentText: String = "",
+    confirmText: String = "",
+    dismissText: String = "",
+    onDismissRequest: () -> Unit = {},
+    onConfirmRequest: () -> Unit = {},
+    content: (@Composable ColumnScope.() -> Unit)? = null
+) {
+    Dialog(
+        onDismissRequest = { onDismissRequest() }
+    ) {
+        Card(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(Paddings.xlarge),
+            shape = RoundedCornerShape(16.dp),
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .background(MaterialTheme.colorScheme.background)
+                    .padding(Paddings.extra),
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                Text(
+                    modifier = Modifier.fillMaxWidth(),
+                    text = titleText,
+                    style = MaterialTheme.typography.titleSmall
+                )
+                Spacer(modifier = Modifier.size(16.dp))
+                Text(
+                    modifier = Modifier.fillMaxWidth(),
+                    text = contentText,
+                    style = MaterialTheme.typography.labelMedium.copy(
```

Comment:
🚫 **[ktlint]** <standard:multiline-expression-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A multiline expression should start on a new line
<!-- __reviewdog__:ChAzZWQyYjliN2QyY2NhYzY3EgZrdGxpbnQ= -->